data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
+			"Our ships can beat anything you stupid monkeys can make."
+			"This sure beats being a factory worker."
+			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
+			"What are you looking at, monkey?"
+			"I can take on a human like you any day."
+			"You looking for a fight, monkey?"
+		word
+			`"`
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."
+			"I feel like the only Hai on this planet who regrets joining the Unfettered."
+			"I want to go home."
+		word
+			`"`
+news "unfettered hai after wanderers invasion"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"We shall conquer the worlds to the north as is our right. It is inevitable that the strong shall defeat the weak."
+			"I have fought with the stupid birds to the north. They have no knowledge of proper battle. Our war with them will be swift."
+			"Those overgrown birds have fantastic weaponry, yet they are too cowardly to use it."
+			"I wonder why the birds do not simply fight back. Their weapons are clearly capable of doing so."
+			"I wish I could get my hands on some of that Wanderer tech."
+			"Wanderer worlds are pristine! We have won a great prize in our battle against the stupid birds."
+			"The true Hai will no longer be confined in their Drak-imposed cage."
+			"I'll see great glory in fighting the weak-willed birds."
+			"The worlds up north are nothing like the ones we were confined to. On the Wanderer worlds, the air is clean and the water is as clear as glass."
+		word
+			`"`
+news "unfettered hai after wanderers invasion regret"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"I had to shoot down a Wanderer ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
+			"The Wanderers hardly try to fight back. It makes me wonder if they want us to conquer their planets."
+			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
+			"I came here to do battle with warriors, not with merchants and cowardly birds."
+			"What challenge is there in killing the weak? I desire to battle with those who are worthy."
+			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and threaten to kill me."
```

Comment:
Again, actual narrative so far suggests that killing each other isn't done often or personally.
```suggestion
			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and tell me to ship myself home"
```